worlds/zork_grand_inquisitor/enums.py

```diff
@@ -0,0 +1,350 @@
+import enum
```

Comment:
This is mostly a stylistic choice. I prefer to always know explicitly from which other module a callable or foreign classes are from in my code. It can lead to things that seem sillier like this, but I'm super consistent about it. I'm well aware it tends to annoy people 😄